ui/src/components/directory/viewer/set/CompareSetSection.vue

```diff
@@ -0,0 +1,185 @@
+          @mouseover="showOverlay($event, option['@id'])"
+          @mouseleave="hideOverlay($event)"
+        >
+          {{ option.name }} |
+          <span
+            class="member-code"
+            v-tooltip.right="'Copy to clipboard'"
+            v-clipboard:copy="copyToClipboard(option.code)"
+            v-clipboard:success="onCopy"
+            v-clipboard:error="onCopyError"
+            >{{ option.code }}</span
+          >
+        </div>
+      </template>
+    </Listbox>
+    <OverlaySummary ref="OS" />
+  </div>
+</template>
+
+<script setup lang="ts">
+import OverlaySummary from "@/components/shared/OverlaySummary.vue";
+import setupOverlay from "@/composables/setupOverlay";
+import { DirectService, EntityService } from "@/services";
+import { useFilterStore } from "@/stores/filterStore";
+import { SortDirection, ToastSeverity } from "@im-library/enums";
+import { isArrayHasLength, isObject, isObjectHasKeys } from "@im-library/helpers/DataTypeCheckers";
+import { FilterOptions } from "@im-library/interfaces";
+import { SearchRequest, SearchResultSummary, TTIriRef } from "@im-library/interfaces/AutoGen";
+import { ToastOptions } from "@im-library/models";
+import { IM, RDFS } from "@im-library/vocabulary";
+import { useToast } from "primevue/usetoast";
+import { ComputedRef, Ref, computed, onMounted, ref, watch } from "vue";
+interface Props {
+  header: string;
+  selectedSet?: SearchResultSummary;
+  members: any[];
+  setIri?: string;
+  loading: boolean;
+}
+const props = defineProps<Props>();
+
+const toast = useToast();
+const directService = new DirectService();
+const { OS, showOverlay, hideOverlay } = setupOverlay();
+const filterStore = useFilterStore();
+const storeSelectedFilters: ComputedRef<FilterOptions> = computed(() => filterStore.selectedFilters);
+const selectedFilters: Ref<FilterOptions> = ref({ ...storeSelectedFilters.value });
+const controller: Ref<AbortController> = ref({} as AbortController);
+
+const selectedSet: Ref<SearchResultSummary> = ref({} as SearchResultSummary);
+const filteredSets: Ref<SearchResultSummary[]> = ref([]);
+const selected = ref();
+
+const emit = defineEmits({ "update:selectedSet": _payload => true });
+
+watch(
+  () => selectedSet.value.iri,
+  async newValue => emit("update:selectedSet", selectedSet.value)
+);
+
+onMounted(async () => {
+  await init();
+});
+
+async function init() {
+  if (props.setIri) {
+    const entity = await EntityService.getPartialEntity(props.setIri, [RDFS.LABEL]);
+    selectedSet.value = { iri: entity["@id"], name: entity[RDFS.LABEL] } as SearchResultSummary;
+  }
+}
+
+function copyToClipboard(code: string) {
+  return code;
+}
+
+function onCopy(): void {
+  toast.add(new ToastOptions(ToastSeverity.SUCCESS, "Code copied to clipboard"));
+}
+
+function onCopyError(): void {
+  toast.add(new ToastOptions(ToastSeverity.ERROR, "Failed to copy code to clipboard"));
+}
+
+async function search(searchText: string): Promise<SearchResultSummary[]> {
```

Comment:
this method probably needs updating to handle paging, or ui needs updating to state only first 100 results available

Reply:
Addressed. Only first 100 results are being returned.